presto-main/src/main/java/io/prestosql/server/security/AuthenticationFilter.java

```diff
@@ -41,14 +41,20 @@
     private final List<Authenticator> authenticators;
```

Comment:
can you please add some unit tests for this feature?

Reply:
I updated the existing test in `TestResourceSecurity`